### PR TITLE
Package ocamline.1.0

### PR DESCRIPTION
**Pull request by chrisnevers**
### `ocamline.1.0`
Command line interface for user input

---
* Homepage: https://github.com/chrisnevers/ocamline
* Source repo: git://github.com/chrisnevers/ocamline.git
* Bug tracker: https://github.com/chrisnevers/ocamline/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
There is no `url` section in the opam file. My guess is that you forgot to push the git tag before running `opam publish`

File: packages/ocamline/ocamline.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+bug-reports: "https://github.com/chrisnevers/ocamline/issues"
+dev-repo: "git://github.com/chrisnevers/ocamline.git"
+synopsis: "Command line interface for user input"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/ocamline/ocamline.1.0/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Dune should not be tagged as a build dependency (see #14266), moreover, your `dune-project` file says this package requires at least dune 1.9

**Comment by kit-ty-kate** [comment]
This should be good to merge once my comment above has been taken care of.

File: packages/ocamline/ocamline.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+homepage: "https://github.com/chrisnevers/ocamline"
+bug-reports: "https://github.com/chrisnevers/ocamline/issues"
+dev-repo: "git://github.com/chrisnevers/ocamline.git"
+synopsis: "Command line interface for user input"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
```
Comment (kit-ty-kate) [inline packages/ocamline/ocamline.1.0/opam]:
```suggestion
depends: [
  "ocaml"
```
Oh, also this should be there somewhere

**Comment by chrisnevers** [comment]
@kit-ty-kate thank you for the help!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**